service/storage_service.cc

```diff
@@ -3769,6 +3777,7 @@ future<> storage_service::replicate_to_all_cores(mutable_token_metadata_ptr tmpt
 }
 
 future<> storage_service::stop() {
+    co_await _task_manager_module->stop();
```

Comment:
Shouldn't we stop the t.m. module after aborting the node ops, as per the line below?
Or is stopping the module does that via the tasks interface, it should replace the below.